playbooks/ha_openlabcmd_install.yaml

```diff
@@ -18,6 +18,12 @@
 
     - name: Init HA node
       shell: |
+        node=`openlab ha node list --role '{{ node_role }}' --type  '{{ node_type }}'`
+
+        if node
+           openlab ha node delete node
+        fi
+
```

Comment:
for  i in `openlab ha node list | awk '{print $2}' | grep -`; do openlab ha node delete $i ;done